src/Components/UserProfilePage/Boxes/MissionSummary.tsx

```diff
@@ -0,0 +1,51 @@
+import { Card, Divider, Typography } from "@mui/material";
+import MissionResponse from "~/types/Response/MissionResponse";
+interface MissionSummaryProps {
+  mission?: MissionResponse;
+}
+const MissionSummary = ({ mission }: MissionSummaryProps) => {
+  if (!mission) {
+    return <>no missions</>;
+  }
+  const completedChores = mission.missionChores.filter((m) => m.completed);
+  const incompleteChores = mission.missionChores.filter((m) => !m.completed);
+  let incompleteChoresPoints = 0;
+  for (const chore of incompleteChores) {
+    incompleteChoresPoints += chore.points;
+  }
+  let completedChoresPoints = 0;
+  for (const chore of completedChores) {
+    completedChoresPoints += chore.points;
+  }
+  return (
+    <Card sx={{ minWidth: 275 }}>
+      <Typography variant="h6">Last Mission Summary</Typography>
+      <Divider />
+      <Typography variant="body1">
+        Mission Number:{(mission.missionId ?? 0) + 1}
```

Comment:
I'm not sure this will work as expected. Missions for all users are stored in the same table. So I believe a mission could have a missionID == 12, but for that user, it is their first mission.  Instead I think we should be able to use the length of  userData.missions

As of now users can't delete a mission, so the last mission will have the same number as the length of the missions list. You could calculate this in the UserProfileView and just pass the number to MissionSummary so you don't have to pass in as much data.

```suggestion
        Mission Number:{userData.mission}
```